app/src/main/dist/config/crate.yml

```diff
@@ -34,9 +34,8 @@
 #gateway.recover_after_nodes: 2
 #discovery.zen.minimum_master_nodes: 2
 
-# most cloud environments don't support multicast discovery
+# Specify the instances that you want to form a cluster
```

Comment:
this is anyways a duplicate, see further down. i think it would make sense to look at the file as a whole and consider rephrasing some stuff, because if the multicast is removed, then actually unicast is THE discovery mechanism for non-cloud environments. so let's say the only standalone way to do discovery